templates/default.yaml

```diff
@@ -565,7 +565,7 @@ propagateProxyEnv: null
 # can only work when using a vmnet network interface and the host has working
 # IPv6 configured as well.
 hostResolver:
-  # 🟢 Builtin default: true
+  # 🟢 Builtin default: true (except for vmType: vz, where it is false)
```

Comment:
https://lima-vm.io/docs/config/network/user/#dns-19216853 needs to be updated to explain why this is not needed on vz?